### DISCUSSION TITLE
How do i blacklist a folder from being searched?

### DISCUSSION THREAD

**Discussion by NastyFlytrap** [deckerst/aves#989] [General]
I have a folder that has a bunch of DOS games and their assets in them. I dont want Aves Libre to search this folder because its just going to ruin my collection view with a bunch of crap. I put a .nomedia file in the folder already and it didnt care.

**Comment by deckerst** [comment]
No need for the .nomedia file. See [this](https://github.com/deckerst/aves/issues/13#issuecomment-2066437678) instead.

**Reply by NastyFlytrap** [reply]
Alright, i was just coming from this thread here, thats where i got that from. https://github.com/deckerst/aves/issues/13#issuecomment-2066442097

Thanks!
